src/_config.ts

```diff
@@ -27,6 +27,17 @@ export function semanticReleaseGit(assets: string[]): SemanticReleasePlugin {
   ];
 }
 
+export function semanticReleaseOpenApi(
+  apiSpecFiles: string[]
+): SemanticReleasePlugin {
+  return [
+    "@aensley/semantic-release-openapi",
```

Comment:
You need to add this package to the `package.json`: `npm install @aensley/semantic-release-openapi`